Description: Ignore these tests because the file they use are missing
 The files are not in the orig.tar
 Test would fail with:
  0: IO Error
  1: No such file or directory (os error 2)
Forwarded: not-needed
Last-Update: 2023-03-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/integration.rs
+++ b/tests/integration.rs
@@ -19,6 +19,7 @@
     )
 }

+#[ignore]
 #[test]
 fn open_file() {
     let _ = run!(File::open(file_path!("example.pdf")));
@@ -120,6 +121,7 @@
     }
 }

+#[ignore]
 #[test]
 fn parse_objects_from_stream() {
     use pdf::object::NoResolve;
